src/repositories/habit.repository.js

```
@@ -1,5 +1,6 @@
 const { dbAll, dbGet, dbRun } = require('../utils/dbUtils');
 const { DatabaseError } = require('../utils/errors');
```

ESLint

[error] Missing file extension for "../utils/errors"

(import/extensions)